graphql-server/Tests/AppTests/AppTests.swift

```diff
@@ -0,0 +1,42 @@
+import App
+import Hummingbird
+import HummingbirdXCT
+import XCTest
+
+final class AppTests: XCTestCase {
+    static let allTests = [
+        ("testGraphQLSuccess", testGraphQLSuccess),
+    ]
+    func testGraphQLSuccess() throws {
+        let expectation = XCTestExpectation(description: "GraphQL")
+        expectation.expectedFulfillmentCount = 3
+        
```

Comment:
I haven't used `XCTestExpectation` much but it tends to be used when waiting on async results. `XCTExecute` does that wait for you. Plus you aren't actually testing to see if the expectation is fulfilled. I can remove a `expectation.fulfill()` line and the test is still successful.

Reply:
Done